doc/releases/release-notes-4.0.rst

```diff
@@ -24,6 +24,17 @@ https://docs.zephyrproject.org/latest/security/vulnerabilities.html
 API Changes
 ***********
 
+* Removed deprecated arch-level CMSIS header files
+  `include/zephyr/arch/arm/cortex_a_r/cmsis.h` and
+  `include/zephyr/arch/arm/cortex_m/cmsis.h`. `cmsis_core.h` needs to be
+  included now.
+
+* Removed deprecated `ceiling_fraction` macro. :c:macro:`DIV_ROUND_UP` needs to
+  be used now.
+
+* `EARLY`, `APPLICATION` and `SMP` init levels can no longer be used for
+  devices.
```

Comment:
```suggestion
* Removed deprecated arch-level CMSIS header files
  ``include/zephyr/arch/arm/cortex_a_r/cmsis.h`` and
  ``include/zephyr/arch/arm/cortex_m/cmsis.h``. ``cmsis_core.h`` needs to be
  included now.

* Removed deprecated ``ceiling_fraction`` macro. :c:macro:`DIV_ROUND_UP` needs to
  be used now.

* ``EARLY``, ``APPLICATION`` and ``SMP`` init levels can no longer be used for
  devices.
```

Reply:
almost tempted to ban single backticks in checkpatch or something, pretty sure there is an equivalent role if people really want to achieve italics

Reply:
Thanks for catching, I keep getting hit by this...